presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -298,7 +298,7 @@ public WriteMapping toWriteMapping(ConnectorSession session, Type type)
             return WriteMapping.longMapping("timestamp", timestampWriteFunction(session));
         }
         if (TIMESTAMP_WITH_TIME_ZONE.equals(type)) {
-            return WriteMapping.longMapping("timestamp with time zone", timestampWithTimeZoneWriteFunction());
+            return WriteMapping.longMapping("timestamptz", timestampWithTimeZoneWriteFunction());
```

Comment:
Do i understand correctly that array of `timestamp with time zone` needs to be declared as `timestamptz[]` in pg?

Reply:
It's possible to create arrays with `timestamp with time zone[]` but it's not possible to do it with `PgConnection#createArrayOf` that is used by `PostgreSqlClient#arrayWriteFunction`